qa/wildfly-runtime/src/main/wildfly/domain/configuration/host.xml

```diff
@@ -54,7 +54,7 @@
         <management-interfaces>
             <http-interface security-realm="ManagementRealm">
                 <http-upgrade enabled="true"/>
-                <socket interface="management" port="${jboss.management-http.port}"/>
+                <socket interface="management" port="${jboss.management.http.port:9990}"/>
```

Comment:
Revert this change.

Reply:
Het @langfr,

Thanks for the input, that was it!

Best,
Nikola